src/lib/LibProcessStream.sol

```diff
@@ -4,6 +4,8 @@ pragma solidity =0.8.25;
 import {Vm} from "forge-std/Vm.sol";
 import {Strings} from "openzeppelin-contracts/contracts/utils/Strings.sol";
 import {InputStream} from "lib/sushiswap/protocols/route-processor/contracts/InputStream.sol";
+import {IUniswapV2Pair} from "lib/sushiswap/protocols/route-processor/interfaces/IUniswapV2Pair.sol";
+
 import "forge-std/console2.sol";
 
 /// Library to decompile sushi routes
```

Comment:
should we somehow indicate which route processor this is all compatible for?

Reply:
How do i do that ?

Reply:
Updated